frontend/src/app/corpus-definitions/form/meta-form/meta-form.component.html

```diff
@@ -0,0 +1,46 @@
+<h1 class="title">Corpus Metadata</h1>
+<h2 class="subtitle">Provide some basic information about your corpus</h2>
+<form (ngSubmit)="onSubmit()" [formGroup]="metaForm">
+
+        <div class="field">
+            <label class="label" for="title">Title</label>
+            <div class="control">
+                <input class="input" type="text" name="title" formControlName="title" />
+            </div>
+            <p class="help">Enter a short, clear title for your corpus.</p>
+        </div>
+
+        <div class="field">
+            <label class="label" for="description">Description</label>
+            <div class="control">
+                <input class="input" type="text" name="description" formControlName="description" />
+            </div>
+        </div>
+
+        <div class="field">
+            <label class="label" for="category">Category</label>
+            <div class="control">
+            <p-dropdown [options]="categories" formControlName="category" placeholder="select category" />
+            </div>
+        </div>
+
+        <div class="field is-grouped">
+            <div formGroupName="date_range">
+                <label class="label" for="min">Start date</label>
+                <div class="control">
+                    <input class="input" type="date" name="min" formControlName="min" />
+                </div>
+                <label class="label" for="max">End date</label>
+                <div class="control">
+                    <input class="input" type="date" name="max" formControlName="max" />
+                </div>
+                <p class="help">Enter the earliest and latest date in your corpus in YYYY-MM-DD format</p>
+            </div>
+        </div>
```

Comment:
Since we only use the year, you could make this a numeric control and just add January 1st for the min date, and December 31st for the max date.

In the help text, we could also mention that this doesn't need to be an exact year, it's just an indication for users.